src/index.html

```diff
@@ -6,9 +6,69 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;0,700;1,500;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="img" href="#">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo">
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              class="link is-active" href="#1">
```

Comment:
```suggestion
              class="link is-active" href="#aplle">
```
краще використовувати назви, ніж цифри